docs/api/permission.rst

```diff
@@ -115,16 +116,18 @@ and its title is shortened:
 
 .. doctest::
 
-   >>> vocab.getTermByToken('zope.Public').title
-   'Public'
+   >>> print(vocab.getTermByToken('zope.Public').title)
+   Public
 
 The terms are sorted by title except for the public permission, which is
 listed first:
 
 .. doctest::
 
-   >>> [term.title for term in vocab]
-   ['Public', 'x', 'y']
+   >>> for t in [term.title for term in vocab]: print(t)
```

Comment:
This looks like a mechanical change.  I would prefer `>>> for term in vocab: print(term.title)`.

Reply:
OK.